pkg/search/options.go

```diff
@@ -240,7 +240,10 @@ var (
 	ComplianceOperatorProfileRef             = newFieldLabel("Profile Ref ID")
 	ComplianceOperatorScanRef                = newFieldLabel("Scan Ref ID")
 	ComplianceOperatorRuleRef                = newFieldLabel("Rule Ref ID")
+	ComplianceOperatorRuleId                 = newFieldLabel("Compliance Rule ID")
```

Comment:
Are these changes supposed to be on top of another PR?  Seems like the corresponding items in the storage proto are missing.

Reply:
This field is automatically added as a search field to the database. It does not need to be part of the rules storage proto.
Initially I had use it in my query but removed it to get distinct results. We'd like to keep it here that in the future someone can select the rule id which was not possible before.

Reply:
Removed it.